s3transfer/crt.py

```diff
@@ -150,6 +159,11 @@ def create_s3_crt_client(
 
 
 class CRTTransferManager:
+
+    ALLOWED_DOWNLOAD_ARGS = TransferManager.ALLOWED_DOWNLOAD_ARGS
+    ALLOWED_UPLOAD_ARGS = TransferManager.ALLOWED_UPLOAD_ARGS
+    ALLOWED_DELETE_ARGS = TransferManager.ALLOWED_DELETE_ARGS
```

Comment:
Should I make a common base class for `TransferManager` and `CRTTransferManager`, where I can put stuff like these lists, and `def _validate_all_known_args()` and `def _validate_if_bucket_supported()`?

Or move these to be standalone lists/functions that both classes can use?

Or just copy/paste the functions, and reference the lists, like I'm doing here?